### DISCUSSION TITLE
Add Nexus protocol for temperature/humidity sensors to rc-switch library

### DISCUSSION THREAD

**Discussion by joejoe204** [arendst/Tasmota#11071] [Ideas]
Outdoor temperature/humidity sensors transmitting the data at 433MHz using the Nexus protocol are quite popular. I extended the rc-switch library to process the signals in tasmota. Detailed analysis of the data can easily be done in the home automation (https://community.openhab.org/t/how-to-receive-temperature-and-humidity-from-433-mhz-outdoor-sensor/117524).
How about adding the protocol to rc-switch library:
RCSwitch.cpp
`{ 500,  0, { 0, 0 }, 1, {   0,  7 }, { 1,  2 },  { 1, 4}, false,  0 }   // 36 (Nexus, 36 bit)`

increase receive buffer to 40 bit:
`#define RCSWITCH_MAX_CHANGES 83`

**Comment by barbudor** [comment]
Sorry, do you say you've made a PR to sui77's rcswitch library? 
I can't find it
I believe we want to limit drift between sui77's original and tasmotas fork. So if you have additional protocols to support it would be better to have them 1st in the original. 
Apologies if I didn't understood well

**Reply by joejoe204** [reply]
You are right, in general it is better to change the origin.  Tasmota is using the fork `https://github.com/1technophile/rc-switch/releases/latest`. I noticed that a protocol was already added to Tasmota's rc-switch library and that's the reason why I asked.

**Comment by Jason2866** [comment]
Providing a PR to Tasmota RC-Switch library is okay. We have a modified version of `https://github.com/1technophile/rc-switch/releases/latest` since we had to remove stuff which is redundant/conflicting and added protocols and functions which are not in the fork and will never be there because it is Tasmota specific.
EDIT: PR #11099
